releng/helm-charts/xsk-kpack/templates/kpack.yaml

```diff
@@ -21,14 +29,16 @@ apiVersion: kpack.io/v1alpha1
 kind: ClusterStore
 metadata:
   name: kneo-xsk-store
+  namespace: {{ .Values.create.namespace }}
 spec:
   sources:
-  - image: "{{ .Values.imageBuilder.buildpack }}:#{XSKVersion}#"
+  - image: "{{ .Values.imageBuilder.buildpack }}:0.11.0"
```

Comment:
Shouldn't it be `#{XSKVersion}#` instead of hardcoded version `0.11.0`?

Reply:
Yes I saw this. Thanks!